openbas-front/src/components/common/queryable/pagination/PaginationComponentV2.tsx

```diff
@@ -69,11 +69,6 @@ const PaginationComponentV2 = <T extends object>({
   const [options, setOptions] = useState<OptionPropertySchema[]>([]);
 
   useEffect(() => {
-    // Retrieve input from uri
-    if (queryableHelpers.uriHelpers) {
-      queryableHelpers.uriHelpers.retrieveFromUri();
-    }
-
```

Comment:
Deleted from here because we do it directly in useQueryableWithLocalStorage.tsx now